src/libraries/System.Private.CoreLib/src/System/Threading/Timer.cs

```diff
@@ -103,7 +103,15 @@ public TimerQueueDebuggerTypeProxy(TimerQueue queue)
             }
 
             [DebuggerBrowsable(DebuggerBrowsableState.RootHidden)]
-            public TimerQueueTimer[] Items => new List<TimerQueueTimer>(_queue.GetTimersForDebugger()).ToArray();
+            public TimerQueueTimer[] Items
```

Comment:
This is in a debugger type proxy.... the extra allocations here don't matter.... better to keep the code simpler.